src/core_ocean/Registry.xml

```diff
@@ -1582,6 +1607,16 @@
 			<var name="evaporationFlux"/>
 			<var name="rainFlux"/>
 		</stream>
+		<stream name="atmospheric_forcing"
+				type="input"
+				filename_template="atmospheric_forcing.nc"
+				filename_interval="none"
+				input_interval="none"
+				immutable="true">
```

Comment:
I could be wrong, but wouldn't it make sense to have a package for forcing (and maybe for atmospheric forcing specifically)? @pwolfram?